tests/api-testing/tests/test_pipeline.py

```
@@ -342,3 +342,765 @@ def test_pipeline_creation_and_action_with_schedule(create_session, base_url, pi
+
+
+
+@pytest.mark.parametrize(
+    "pipeline_name, stream_name, action, expected_status",
+    [
+        ("pipeline12222", "stream_pytest_data", None, 400),  # No action (default behavior)
+     
+    ]
+)
+ 
+def test_pipeline_creation_and_action_realtime(create_session, base_url, pipeline_name, stream_name, action, expected_status):
```

Ruff (0.8.2)

Redefinition of unused `test_pipeline_creation_and_action_realtime` from line 11

(F811)